docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,375 @@
 ---
-title: "Configuration"
+title: "Spring Config Server"
 ---
 
-Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized
-configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications
-across all environments.
+Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications across all environments.
 
-The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, and it is configured to store the
-configuration in the Oracle Autonomous Database, so it easily supports labelled versions of configuration
-environments, as well as being accessible to a wide range of tools for managing the content.
-Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
+The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, configured to store the Configuration in the Oracle Autonomous Database to support labeled versions of configuration environments as well as being accessible to a wide range of tools for managing the content. More details can be found here: ([Spring Cloud Config Documentation](https://spring.io/projects/spring-cloud-config)).
 
-For example, the Spring Cloud Config client's Spring `application.yaml` configuration file could include:
+When building applications using Spring Config Server the Spring Cloud Config Client's `application.yaml` configuration file must include to access information the deployed Spring Config Server:
```

Comment:
You need a comma after "Spring Config Server"

Change "Client's" to "client's"

Change "... include to access information the ..." to "... include access to information in the ..."

Reply:
Fixed